src/app/ui/universal-dapp-ui.js

```diff
@@ -177,16 +177,22 @@ UniversalDAppUI.prototype.renderInstanceFromABI = function (contractABI, address
       }
     }
     if (calldata) {
-      if (calldata.length > 3 && calldata.substr(0, 2) === '0x') {
+      if (calldata.length < 2) setLLIError('the calldata should be a valid hexadecimal value with size of at least one byte.')
+      if (calldata.length < 4 && calldata.substr(0, 2) === '0x') {
```

Comment:
this is more than that, the size should be even

Reply:
:) thanks